README.md

```diff
@@ -43,6 +43,18 @@ or
 pip install stac_fastapi.opensearch
 ```
 
+### To install and run via pre-built Docker Images
+
+We provide ready-to-use Docker images through GitHub Container Registry ([ElasticSearch](https://github.com/stac-utils/stac-fastapi-elasticsearch-opensearch/pkgs/container/stac-fastapi-es) and [OpenSearch](https://github.com/stac-utils/stac-fastapi-elasticsearch-opensearch/pkgs/container/stac-fastapi-os) backends). You can easily pull and run these images:
+
```

Comment:
Getting close. Do we need to explain how you would connect the db to one of these instances?

Reply:
I think it's clear with using an .env file